docs/conf.py

```diff
@@ -15,12 +15,31 @@
 import sys
 import os
 import shlex
+from mock import Mock as MagicMock
```

Comment:
This is confusing because there is already a ``mock.MagicMock`` class that is distinct from ``mock.Mock``.